Add zh-TW localization

auroursa opened the pull request:
Traditional Chinese is widely utilized in Taiwan, Hong Kong, and Macau, boasting a user base of nearly 30 million globally. For this project, I took the traditional Chinese from Taiwan as the basis for creating the messages.po file.
I humbly seek additional feedback from the community users, thank you.

auroursa commented:
Thanks to @KuwaLee and @noeFly for their contributions to the translation. I'm not sure why "Lint / Run linters (pull_request)" is throwing an error, but I believe we are ready to go.

snowleo208 commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4527 @@
+
+#: src/view/screens/AppPasswords.tsx:195
+msgid "Use app passwords to login to other Bluesky clients without giving full access to your account or password."
+msgstr "使用應用程式專用密碼登錄到其他 Bluesky 客戶端，而無需提供對您的帳號或密碼的完全訪問權限。"
```
```suggestion
msgstr "使用應用程式專用密碼登錄到其他 Bluesky 客戶端，而無需提供對你的帳號或密碼的完全訪問權限。"
```
`你`

snowleo208 commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4527 @@
+
+#: src/screens/Deactivated.tsx:134
+msgid "We estimate {estimatedTime} until your account is ready."
+msgstr "我們估計還需要 {estimatedTime} 才能完成你的帳號準備。"
```
```suggestion
msgstr "我們估計還需要 {estimatedTime} 才能準備好你的帳號。"
```

auroursa commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4527 @@
+
+#: src/view/screens/Settings.tsx:650
+#~ msgid "App passwords"
+#~ msgstr "應用程式專用密碼"
```
是的，因为前面用 #~ 注释掉了

auroursa commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4527 @@
+
+#: src/view/screens/PreferencesHomeFeed.tsx:111
+msgid "Fine-tune the content you see on your home screen."
+msgstr "微調你在首頁上所看到的內容。"
```
我觉得改善可能会更好

auroursa commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4527 @@
+
+#: src/App.native.tsx:60
+msgid "Sorry! Your session expired. Please log in again."
+msgstr "抱歉！您的會話已過期。請重新登入。"
```
这个我是参考简体常用的 session 会话 来翻译，但搜了下确实在台湾比较少用會話
登入 确实更好理解，也和下句 請重新登入 能对应上

auroursa commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4527 @@
+
+#: src/view/com/posts/FeedErrorMessage.tsx:114
+msgid "This feed is currently receiving high traffic and is temporarily unavailable. Please try again later."
+msgstr "該訊息流目前使用人數較多，服務暫時不可用。請稍後再試。"
```
我这里将 不能 改为 無法，与 3877 行 無法查看此內容 对应上

auroursa commented:
Thanks to @snowleo208 for the review. I have modified the translation wording based on the suggestions and replaced all instances of "您" with "你" throughout the entire text.
Thanks again for your hard work and contribution.

noeFly commented:
```po
#: src/view/com/modals/VerifyEmail.tsx:214
msgid "I have a code"
msgstr "我有邀請碼"
```
約 [L1740](https://github.com/auroursa/bluesky-social-app/blob/7dedb074484c9c1df5f15c32d368d34963b2ea8d/src/locale/locales/zh-TW/messages.po#L1740) 處，依照上方參照檔案的檔名看來，這個字串的用途是驗證電子郵件相關，故不太確定是否要將此處的「code」翻譯更改為「驗證碼」。
各位譯者覺得此處應該翻譯成「邀請碼」還是「驗證碼」？

kisaragi-hiu commented:
這種時候應該用「驗證碼」(這是在 VerifyEmail.tsx 裡的)；看來源檔案這個字串沒有在其他地方用到，所以應該不會有問題。(另外可以利用 GitHub 的 code review 功能，會比較好追蹤已處理的問題點)

snowleo208 commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -2688,15 +2688,15 @@ msgstr "播放 GIF"
 #: src/view/com/auth/create/state.ts:177
 msgid "Please choose your handle."
-msgstr "請選擇您的使用者識別符。"
+msgstr "請請選擇你的使用者識別符。"
```
```suggestion
msgstr "請選擇你的使用者識別符。"
```

snowleo208 commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -4072,7 +4072,7 @@ msgstr "上傳文字檔案至："
 #: src/view/screens/AppPasswords.tsx:195
 msgid "Use app passwords to login to other Bluesky clients without giving full access to your account or password."
-msgstr "使用應用程式專用密碼登錄到其他 Bluesky 客戶端，而無需提供對您的帳號或密碼的完全訪問權限。"
+msgstr "使用應用程式專用密碼登錄到其他 Bluesky 客戶端，而無需提供對你的帳號或密碼的完全訪問權限。"
```
我在想這句不用這麼長，只是想說用app password就不用提供帳戶與密碼吧

```suggestion
msgstr "使用應用程式專用密碼登錄到其他 Bluesky 客戶端，而無需提供你的帳號或密碼。"
```

snowleo208 commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -4399,7 +4399,7 @@ msgstr "你還沒有封鎖任何帳號。要封鎖帳號，請轉到其個人資
 #: src/view/screens/AppPasswords.tsx:87
 msgid "You have not created any app passwords yet. You can create one by pressing the button below."
-msgstr "你還沒有建立任何應用程式專用密碼。您可以通過按下下面的按鈕來創建一個。"
+msgstr "你還沒有建立任何應用程式專用密碼。你可以通過按下下面的按鈕來創建一個。"
```
```suggestion
msgstr "你還沒有建立任何應用程式專用密碼，如你想建立一個，按下面的按鈕。"
```

這句修改得自然少許

snowleo208 reviewed (commented):
謝謝你的修改，我再加了少許意見 :+1:

noeFly commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4527 @@
+
+#: src/view/com/modals/DeleteAccount.tsx:111
+msgid "For security reasons, we'll need to send a confirmation code to your email address."
+msgstr "為了保護你的帳戶安全，我們需要將驗證碼發送到你的電子郵件地址。"
```
建議統一譯名，避免使用者混亂。

noeFly replied:
統一譯名，避免造成使用者混亂。
```suggestion
msgstr "為了保護你的帳號安全，我們需要將驗證碼發送到你的電子郵件地址。"
```

snowleo208 commented:
> Thanks to @KuwaLee and @noeFly for their contributions to the translation. I'm not sure why "Lint / Run linters (pull_request)" is throwing an error, but I believe we are ready to go.

Someone changed the `src/view/com/composer/text-input/TextInput.web.tsx` in the latest commit. I saw there's a fixing PR ready but haven't merged yet :disappointed:

noeFly commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4527 @@
+
+#: src/view/com/modals/AddAppPasswords.tsx:209
+msgid "For security reasons, you won't be able to view this again. If you lose this password, you'll need to generate a new one."
+msgstr "為了保護你的帳戶安全，你將無法再次查看此內容。如果你丟失了此密碼，你將需要生成一個新密碼。"
```
統一譯名，避免造成使用者混亂。
```suggestion
msgstr "為了保護你的帳號安全，你將無法再次查看此內容。如果你丟失了此密碼，你將需要生成一個新密碼。"
```

noeFly reviewed (commented):
統一譯名（帳戶 -> 帳號）避免造成使用者混亂

snowleo208 commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4527 @@
+
+#: src/view/com/posts/FollowingEndOfFeed.tsx:48
+msgid "You've reached the end of your feed! Find some more accounts to follow."
+msgstr "你已經瀏覽完你的訂閱訊息流啦！尋找一些更多的帳號追蹤吧。"
```
```suggestion
msgstr "你已經瀏覽完你的訂閱訊息流啦！追蹤更多的帳號吧。"
```

noeFly commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4527 @@
+
+#: src/view/com/modals/VerifyEmail.tsx:214
+msgid "I have a code"
+msgstr "我有邀請碼"
```
根據上方的參照檔案的檔名，這個語境應該是在驗證電子郵件的代碼。
```suggestion
msgstr "我有驗證碼"
```

snowleo208 commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4527 @@
+
+#: src/view/com/auth/login/SetNewPasswordForm.tsx:107
+msgid "You will receive an email with a \"reset code.\" Enter that code here, then enter your new password."
+msgstr "你將收到一封帶有重置代碼的電子郵件。請在此輸入該重置代碼，然後輸入你的新密碼。"
```
```suggestion
msgstr "你將收到一封包含重置碼的電郵。請在此輸入該重置代碼，然後輸入你的新密碼。"
```
上面用`重置碼`
帶有/附上/包含重置碼？無論怎翻總是覺得怪怪的 :thinking:

snowleo208 replied:
不錯，那就用`包含`吧

snowleo208 commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4527 @@
+
+#: src/view/com/posts/DiscoverFallbackHeader.tsx:29
+msgid "We ran out of posts from your follows. Here's the latest from <0/>."
+msgstr "我們已經看完了你追蹤的貼文。這是來自 <0/> 的最新消息。"
```
```suggestion
msgstr "我們已再找不到更多貼文。這是來自 <0/> 的最新貼文。"
```

Bluesky會幫我看貼文嗎(笑
希望有人再幫我改改這一句

auroursa replied:
「你看完了你追蹤的貼文。這是來自 <0/> 的最新貼文。」
雖然原文是「We」，但改成「你」似乎比較容易理解

snowleo208 replied:
```suggestion
msgstr "你已看完了你追蹤的貼文。這是 <0/> 的最新貼文。"
```

auroursa replied:
我覺得不錯

auroursa commented:
我在上面引用了一個PR，看到一位朋友針對翻譯詞進行了一些修改，例如將「Feeds」翻譯為「河道」，以及將「handle」翻譯為「使用者暱稱」，我希望他能夠參與這邊 的討論，以及使用 review 來共同確定最終的翻譯用詞
I've referenced a PR above where a new contributor made modifications to some translations, such as translating "Feeds" to "河道" and "handle" to "使用者暱稱." I hope they can participate in the discussion here, using the review process to collectively decide on the final translation choices.

SciencesVirus commented:
hi I wanna translate 'feeds' as "河道" and translate 'handle' as "暱稱" . What do you all think?

SciencesVirus commented:
> 我在上面引用了一個PR，看到一位朋友針對翻譯詞進行了一些修改，例如將「Feeds」翻譯為「河道」，以及將「handle」翻譯為「使用者暱稱」，我希望他能夠參與這邊 的討論，以及使用 review 來共同確定最終的翻譯用詞 I've referenced a PR above where a new contributor made modifications to some translations, such as translating "Feeds" to "河道" and "handle" to "使用者暱稱." I hope they can participate in the discussion here, using the review process to collectively decide on the final translation choices.

大家好！ 我想說使用河道可能熟悉推特的臺灣使用者會很親切，而handle原本翻譯為識別符，我覺得有一點點拗口，所以翻譯成暱稱。
造成不便還請見諒 謝謝~

ailan12345 commented:
Feed 感覺翻譯成河道或是時間軸
河道：推特及噗浪用戶比較習慣
時間軸：會與原意有些許差異

SciencesVirus commented:
> @SciencesVirus 我是覺得把`handle`譯成`䁥稱`也不錯，但feeds譯成河道，好像個人色彩重了少許／太playful？ 🤔
> 
> 我想知道推特也是用`河道`的嗎？抱歉，其實我一直只用英文介面…… 我特地在推特轉成中文一看，也找不到哪裏寫著`河道`。可以告訴我在哪裏看到嗎？
> 
> 不少用戶也是由推特轉會到Bluesky(包括我)，我覺得翻譯若是跟推特的也無不可

> @SciencesVirus 我是覺得把`handle`譯成`䁥稱`也不錯，但feeds譯成河道，好像個人色彩重了少許／太playful？ 🤔
> 
> 我想知道推特也是用`河道`的嗎？抱歉，其實我一直只用英文介面…… 我特地在推特轉成中文一看，也找不到哪裏寫著`河道`。可以告訴我在哪裏看到嗎？
> 
> 不少用戶也是由推特轉會到Bluesky(包括我)，我覺得翻譯若是跟推特的也無不可

hi 很抱歉 我表達的不清楚，推特本身好像沒有特別使用河道，但是使用者會使用河道來稱呼feed,噗浪好像也是使用河道來稱呼。

ailan12345 commented:
Feed 感覺翻譯成河道或是時間軸
河道：推特及噗浪用戶比較習慣
時間軸：會與原意有些許差異，但與目前界面用圖相符

auroursa commented:
噗浪和 Twitter 好像都是用「河道」，所以我覺得把 feeds 改成河道比較好理解，但是 handle 確實比較難翻譯，它按[文檔](https://bsky.social/about/blog/4-28-2023-domain-handle-tutorial)的意思是類似於 https://bsky.app/profile/bsky.app 中的 bsky.app 這部分，如果翻譯成「暱稱」的話，可能容易與 display name 或者 username 混淆，可能就不是太合適

SciencesVirus commented:
> 噗浪和 Twitter 好像都是用「河道」，所以我覺得把 feeds 改成河道比較好理解，但是 handle 確實比較難翻譯，它按[文檔](https://bsky.social/about/blog/4-28-2023-domain-handle-tutorial)的意思是類似於 https://bsky.app/profile/bsky.app 中的 bsky.app 這部分，如果翻譯成「暱稱」的話，可能容易與 display name 或者 username 混淆，可能就不是太合適

這個我有注意到，可能可以再想想有沒有更合適的譯法~

snowleo208 commented:
@auroursa 原來如此。`handle`有少許像是uuid的概念，我見uuid是翻譯作`通用唯一辨識碼`（英語：Universally Unique Identifier，縮寫：UUID）
像是`用戶辨識碼`會不會很難懂？

```
Display name = 顯示名稱
Username = 用戶名稱
Handle = 用戶辨識碼
```

唔，還是怪怪的，好像太機械化

或是`稱呼`如何？ :thinking:

SciencesVirus commented:
> > 噗浪和 Twitter 好像都是用「河道」，所以我覺得把 feeds 改成河道比較好理解，但是 handle 確實比較難翻譯，它按[文檔](https://bsky.social/about/blog/4-28-2023-domain-handle-tutorial)的意思是類似於 https://bsky.app/profile/bsky.app 中的 bsky.app 這部分，如果翻譯成「暱稱」的話，可能容易與 display name 或者 username 混淆，可能就不是太合適
> 
> 這個我有注意到，可能可以再想想有沒有更合適的譯法~

剛剛注意到Discord也有使用類似的兩個名稱，他們使用Display name 跟username ，我覺得handle更像username的感覺~

SciencesVirus commented:
> 我在原本的英文文檔裏有時都見到`username`，像是︰
> 
> ```po
> #: src/view/com/auth/login/Login.tsx:99
> msgid "Enter your username and password"
> msgstr "輸入你的使用者名稱和密碼"
> 
> #: src/view/com/auth/login/LoginForm.tsx:115
> msgid "Invalid username or password"
> msgstr "使用者名稱或密碼無效"
> ```
> 
> 大部分都是在登入前見到，唯一有問題的是`ChangeHandle.tsx`是在用Bluesky **username**而不是`handle` 🤔
> 
> ```po
> #: src/view/com/modals/ChangeHandle.tsx:266
> msgid "Sets Bluesky username"
> msgstr "設定 Bluesky 使用者名稱"
> ```
> 
> 但我在Bluesky見到的是Handle︰ ![image](https://private-user-images.githubusercontent.com/13027432/303381837-af5007eb-c81f-4a9b-8a26-a49aa216fc29.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDc0MDU0NjksIm5iZiI6MTcwNzQwNTE2OSwicGF0aCI6Ii8xMzAyNzQzMi8zMDMzODE4MzctYWY1MDA3ZWItYzgxZi00YTliLThhMjYtYTQ5YWEyMTZmYzI5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAyMDglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMjA4VDE1MTI0OVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTRjNGFiZjg3NThjYmUyZDM4YWVkZGIwZDJlNmMxMGI4ZDY5OTRjMWVkMWIzZGQ0YWU4ZDQwZDc3NTNlMmUwMGYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.oECnc4K_nARkgEXy4KKwFwwIfhF1dtL3O1cfyhMOcNM)
> 
> 我猜`username`是在登入前才會出現的名詞，登入後全都是`handle`
> 
> 我是覺得用`用戶名稱`來翻譯也可以啦，不知各位覺得如何

我剛剛新申請了一個帳號，但是也沒有看到使用username，所以我也不知道到底會用在哪裡，我覺得用戶名稱可以~

auroursa commented:
> 我剛剛發現，申請新的帳號時，選擇國際碼的時候，臺灣被當作中國的一省 🙄

There is a PR [#2797](https://github.com/bluesky-social/social-app/pull/2797) dedicated to fixing this issue.

snowleo208 commented:
@SciencesVirus 在登入畫面會見到︰

![image](https://github.com/bluesky-social/social-app/assets/13027432/ef4d38a1-0822-4a64-973c-8cee877a0d70)

SciencesVirus commented:
> @SciencesVirus 在登入畫面會見到︰
> 
> ![image](https://private-user-images.githubusercontent.com/13027432/303383829-ef4d38a1-0822-4a64-973c-8cee877a0d70.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDc0MDU3MTUsIm5iZiI6MTcwNzQwNTQxNSwicGF0aCI6Ii8xMzAyNzQzMi8zMDMzODM4MjktZWY0ZDM4YTEtMDgyMi00YTY0LTk3M2MtOGNlZTg3N2EwZDcwLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAyMDglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMjA4VDE1MTY1NVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTNiMDQyNGE4YzY1ZjA3YTEyZjc3ODVlMThmMTMyOWE5ZThjNTVjNzJjNmU2MThjNzY3ODAzMWNjNzE4ZTIyMTImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.zlEMtstSWw3ZtMdVJooz0Ci5yTKY41hXkXe8nwNsvK8)

但是這邊其實是要輸入handle (?

SciencesVirus commented:
> > 我剛剛發現，申請新的帳號時，選擇國際碼的時候，臺灣被當作中國的一省 🙄
> 
> There is a PR [#2797](https://github.com/bluesky-social/social-app/pull/2797) dedicated to fixing this issue.

THX

noeFly commented:
關於「handle」在 YouTube 中官方翻譯是「帳號代碼」，我覺得這個翻譯不會與「Display Name」和「Username」衝突，也容易理解，各位覺得如何呢？

KuwaLee commented:
@snowleo208 @snowleo208 

![IMG_20240208_232244_770](https://github.com/bluesky-social/social-app/assets/30096511/21c401f1-7aeb-4dde-8644-cf0ed2b07a82)

According to Cambridge Dictionary 
"Handle" is more dedicated noun for name
"Username" usually use as term for "handle"

auroursa commented:
> 關於「handle」在 YouTube 中官方翻譯是「帳號代碼」，我覺得這個翻譯不會與「Display Name」和「Username」衝突，也容易理解，各位覺得如何呢？

I have taken note of this [help page](https://support.google.com/youtube/answer/11585688?hl=zh-Hant), and I agree that「帳號代碼」seems more suitable for translating "handle".

auroursa commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4527 @@
+#: src/view/shell/Drawer.tsx:514
+#: src/view/shell/Drawer.tsx:515
+msgid "Moderation"
+msgstr "管理員"
```
以及我發現這裡不應該翻譯成「管理員」，對照翻譯以後，我發現這裡應該更接近「限制」或類似的詞

pfrazee commented:
Hey all -- love what you're doing here!

I can't read a lot of the discussion. @auroursa since you started the PR, can you let me know when consensus is reached and this PR is ready to go?

auroursa commented:
@noeFly I've decided to translate "handle" as「帳號代碼」and I changed the translation of "Moderation" to「限制」as it seems to better capture the meaning of that feature.
@SciencesVirus I'm currently unsure if translating "Feeds" as「河道」is appropriate, but I want to leave room for discussion. After receiving more feedback from users, we can decide whether to modify it in the next translation file submission.

auroursa commented:
@pfrazee Thank you for your patience. There has been quite a bit of discussion about the translation choices since this is the first time deciding to submit a PR for adding zh-TW translation. However, I believe we are ready to go now.

KuwaLee reviewed (commented):
I believe we are ready to go

kisaragi-hiu commented:
I have some more feedback. I'll put my overall opinions in this comment

- "List" (as in the Twitter concept) should probably be 列表.
- Follow is 跟隨 on Twitter, 追蹤 on Facebook - I think 跟隨 is less jarring for Bluesky.
- "Like" should probably stick to Twitter-esque convention as "喜歡" and not Facebook-like "讚"
- `zh_TW` does not use 自定義 for "custom" or "customize". It is usually "自訂" but can be contextual.
- Access is usually 存取 instead of 訪問 (訪問 is usually used for "visit")
- There is a distinction between 帳號 and 帳戶: the latter is the account while the former can be synonymous with "username". This convention is used by, for example, Google and Twitter. I don't consider this crucial though.
- 捨棄 is more familiar than 丟棄 in `zh_TW`

auroursa commented:
@noeFly @snowleo208 @SciencesVirus @KuwaLee Hello everyone, let's have a final discussion on this PR. Regarding these translation differences, what are your thoughts?

KuwaLee commented:
> @noeFly @snowleo208 @SciencesVirus @KuwaLee Hello everyone, let's have a final discussion on this PR. Regarding these translation differences, what are your thoughts?
1.同意
2.我覺得“追蹤”、“跟隨”都可以
3.同意
4.我覺得自定義也很常用但可能不是正式的
5.同意
6.同意
7.同意

SciencesVirus commented:
> @noeFly @snowleo208 @SciencesVirus @KuwaLee Hello everyone, let's have a final discussion on this PR. Regarding these translation differences, what are your thoughts?

對此我跟@KuwaLee 有一樣的看法~

auroursa commented:
Looks good to me. I agree that the translation terms in Bluesky should be closer to Twitter, given that most users migrated from Twitter. This helps lower the user adaptation threshold. Therefore, I plan to keep it this way for now and discuss further translation modifications in the next PR.

kisaragi-hiu commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4527 @@
+
+#: src/view/com/auth/onboarding/RecommendedFeeds.tsx:57
+msgid "Feeds are created by users to curate content. Choose some feeds that you find interesting."
+msgstr "訊息流由使用者創建並管理。選擇一些你覺得有趣的訊息流。"
```
建立

kisaragi-hiu commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4527 @@
+
+#: src/view/screens/Settings.tsx:280
+msgid "Legacy storage cleared, you need to restart the app now."
+msgstr "舊存儲數據已清除，你需要立即重新啟動應用程式。"
```
儲存資料

kisaragi-hiu commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4527 @@
+
+#: src/Navigation.tsx:198
+msgid "Liked by"
+msgstr "按讚"
```
喜歡

kisaragi-hiu commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4527 @@
+
+#: src/view/com/feeds/FeedSourceCard.tsx:277
+msgid "Liked by {0} {1}"
+msgstr "{0} {1} 按讚"
```
由 {0} {1} 喜歡

kisaragi-hiu reviewed (commented):
This is only half done because I'm off for lunch...

U2FsdGVkX1 commented:
@pan93412

auroursa commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4527 @@
+
+#: src/screens/Onboarding/StepAlgoFeeds/index.tsx:106
+msgid "Custom feeds built by the community bring you new experiences and help you find the content you love."
+msgstr "由社區打造的自定義訊息流帶來新鮮體驗，協助你找到所愛內容。"
```
This is something that requires more time for discussion. Indeed, "訊息流" might be a bit difficult to understand.

auroursa commented:
@kisaragi-hiu I have incorporated the review comments above into a single submission.
This is amazing, and I appreciate your thorough review. I have added your name to the translation team section in the format mentioned in [#2782](https://github.com/bluesky-social/social-app/pull/2782).
Thank you for your contributions to improving the quality of zh-TW translation.

kisaragi-hiu commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4527 @@
+
+#: src/view/com/auth/login/SetNewPasswordForm.tsx:138
+msgid "Input code sent to your email for password reset"
+msgstr "輸入發送到你電子郵箱的重設碼以重置密碼"
```
重設密碼

kisaragi-hiu commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4527 @@
+
+#: src/view/com/auth/login/LoginForm.tsx:236
+msgid "Opens password reset form"
+msgstr "開啟密碼重置表單"
```
重設

kisaragi-hiu commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4527 @@
+#: src/view/com/auth/login/SetNewPasswordForm.tsx:124
+#: src/view/com/auth/login/SetNewPasswordForm.tsx:136
+msgid "Reset code"
+msgstr "重置碼"
```
重設

kisaragi-hiu commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4527 @@
+
+#: src/view/com/modals/ChangePassword.tsx:190
+msgid "Reset Code"
+msgstr "重置碼"
```
重設

kisaragi-hiu commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4527 @@
+
+#: src/view/screens/Settings.tsx:806
+msgid "Reset onboarding"
+msgstr "重置引導流程"
```
```suggestion
msgstr "重設初始設定進行狀態"
```

- same argument for other uses of "onboarding"
- This is only shown for development builds, so in my opinion can be a bit more wordy

kisaragi-hiu commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4527 @@
+
+#: src/view/screens/Settings.tsx:809
+msgid "Reset onboarding state"
+msgstr "重置引導流程狀態"
```
```suggestion
msgstr "重設初始設定進行狀態"
```

Same argument for other uses of "onboarding", though with "進行狀態" to make sure it's not mistaken for "reset to initial settings"

kisaragi-hiu commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4527 @@
+
+#: src/view/com/auth/login/ForgotPasswordForm.tsx:100
+msgid "Reset password"
+msgstr "重置密碼"
```
重設密碼

kisaragi-hiu commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4527 @@
+
+#: src/view/screens/Settings.tsx:796
+msgid "Reset preferences"
+msgstr "重置偏好"
```
```suggestion
msgstr "重設偏好設定"
```

kisaragi-hiu commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4527 @@
+
+#: src/view/screens/Settings.tsx:799
+msgid "Reset preferences state"
+msgstr "重置偏好狀態"
```
```suggestion
msgstr "重設偏好設定狀態"
```

kisaragi-hiu commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4527 @@
+
+#: src/view/screens/Settings.tsx:807
+msgid "Resets the onboarding state"
+msgstr "重置引導流程狀態"
```
```suggestion
msgstr "會重設初始設定狀態"
```

kisaragi-hiu commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4527 @@
+
+#: src/view/screens/Settings.tsx:797
+msgid "Resets the preferences state"
+msgstr "重置偏好狀態"
```
```suggestion
msgstr "會重設偏好設定狀態"
```

kisaragi-hiu commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4527 @@
+
+#: src/view/screens/LanguageSettings.tsx:281
+msgid "Select which languages you want your subscribed feeds to include. If none are selected, all languages will be shown."
+msgstr "選擇你希望訂閱訊息流中所包含的語言。如果未選擇任何語言，將默認顯示所有語言。"
```
I suggest leaving out 默認/預設, or phrasing it like this:

```suggestion
msgstr "選擇你希望訂閱訊息流中所包含的語言。未選擇任何語言時會顯示所有語言。"
```

pan93412 replied:
```suggestion
msgstr "選擇你希望訂閱訊息流中所包含的語言。如果未選擇任何語言，將預設顯示所有語言。"
```

https://tw.hinative.com/questions/13044650

kisaragi-hiu commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4527 @@
+
+#: src/view/screens/LanguageSettings.tsx:98
+msgid "Select your app language for the default text to display in the app"
+msgstr "選擇應用中顯示默認文字的語言"
```
```suggestion
msgstr "選擇應用程式中顯示預設文字的語言"
```

Honestly this string is a bit unclear even in the source text, so this is the best I can do.

- afaik in `zh_TW` App is never shortened to 應用 - it's either 應用程式 or just "app".

pan93412 replied:
```suggestion
msgstr "選擇應用程式中顯示預設文字的語言"
```

In Taiwan, we translate "App" as "應用程式" or "App" (leave it as it is)

kisaragi-hiu commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4527 @@
+
+#: src/view/com/auth/login/ForgotPasswordForm.tsx:118
+msgid "Sets hosting provider for password reset"
+msgstr "設定用於密碼重置的主機提供商資訊"
```
重設

kisaragi-hiu commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4527 @@
+
+#: src/view/com/modals/VerifyEmail.tsx:119
+msgid "This is important in case you ever need to change your email or reset your password."
+msgstr "這很重要，以防你將來需要更改電子郵箱或重置密碼。"
```
重設

kisaragi-hiu commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4527 @@
+
+#: src/view/com/modals/BirthDateSettings.tsx:61
+msgid "This information is not shared with other users."
+msgstr "此信息不與其他使用者共享。"
```
```suggestion
msgstr "此資訊不會分享給其他使用者。"
```

kisaragi-hiu commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4527 @@
+
+#: src/view/com/posts/FeedErrorMessage.tsx:114
+msgid "This feed is currently receiving high traffic and is temporarily unavailable. Please try again later."
+msgstr "該訊息流目前使用人數較多，暫時無法查看。請稍後再試。"
```
```suggestion
msgstr "此訊息流由於目前使用人數眾多而暫時無法使用。請稍後再試。"
```

- 較多 sounds too little
- I think adding a 而 makes the relationship between the two phrases clearer.
- I think 暫時無法使用 makes it more clear that this is a capabilities issue, not just an arbitrary limit, as all uses are unavailable, not just viewing.

kisaragi-hiu commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4527 @@
+#: src/view/screens/ProfileFeed.tsx:475
+#: src/view/screens/ProfileList.tsx:660
+msgid "This feed is empty!"
+msgstr "該訊息流為空！"
```
```suggestion
msgstr "這個訊息流是空的！"
```

More casual.

kisaragi-hiu commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4527 @@
+
+#: src/view/com/posts/CustomFeedEmptyState.tsx:37
+msgid "This feed is empty! You may need to follow more users or tune your language settings."
+msgstr "該訊息流為空！你或許需要先跟隨更多的人或檢查你的語言設定。"
```
```suggestion
msgstr "這個訊息流是空的！你或許需要先跟隨更多的人或檢查你的語言設定。"
```

More casual.

kisaragi-hiu commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4527 @@
+#: src/view/screens/PreferencesThreads.tsx:53
+#: src/view/screens/Settings.tsx:559
+msgid "Thread Preferences"
+msgstr "討論串偏好"
```
I recommend using 對話串 for all "thread", as 討論串 is more for forum threads, not Twitter-esque threads.

auroursa replied:
I plan to maintain consistency by replacing the term "討論串" with "對話串" throughout the entire translation file.

kisaragi-hiu commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4527 @@
+
+#: src/view/com/util/forms/DropdownButton.tsx:246
+msgid "Toggle dropdown"
+msgstr "切換下拉菜單"
```
切換下拉式選單

kisaragi-hiu commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4527 @@
+
+#: src/view/screens/Log.tsx:52
+msgid "View debug entry"
+msgstr "查看除錯入口"
```
```suggestion
msgstr "查看除錯項目"
```

kisaragi-hiu commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4527 @@
+
+#: src/view/screens/Log.tsx:54
+msgid "Opens additional details for a debug entry"
+msgstr "開啟除錯記錄的附加詳細資訊"
```
```suggestion
msgstr "開啟除錯項目的額外詳細資訊"
```

kisaragi-hiu reviewed (commented):
A few major wording choices remaining:

- Repost: 轉貼 not 轉發, for consistency with Twitter. (Though Twitter `zh_TW` still uses 轉推, the 推 refers to Tweet; Twitter and other microblog's convention seems to be 轉推). 轉推 also works as a noun.
- Thread: 對話串 not 討論串, to disambiguate with 討論. Also, 討論串 is more for forum threads, not Twitter-esque chained-post threads.

Other than that, this is looking pretty close to good in my opinion.

kisaragi-hiu commented:
> This is amazing, and I appreciate your thorough review.

@auroursa Thank you too for your work and your fast responses to feedback!

> I have added your name to the translation team section

Thanks :)

auroursa commented:
I have adjusted all the translation based on the review suggestions, and I believe we are ready to go.

kisaragi-hiu commented:
I also think we are ready to go now 😃

pan93412 commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4527 @@
+
+#: src/Navigation.tsx:161
+msgid "People followed by @{0}"
+msgstr "被這些人所跟隨 @{0}"
```
```suggestion
msgstr "被 @{0} 跟隨的人"
```

https://github.com/bluesky-social/social-app/blob/43b447e5f4345bd6c3a6957cb5aa1587324799e8/src/Navigation.tsx#L149-L155

pan93412 commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4527 @@
+
+#: src/Navigation.tsx:154
+msgid "People following @{0}"
+msgstr "正在跟隨 @{0}"
```
```suggestion
msgstr "跟隨 @{0} 的人"
```

https://github.com/bluesky-social/social-app/blob/43b447e5f4345bd6c3a6957cb5aa1587324799e8/src/Navigation.tsx#L156-L162

pan93412 commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4527 @@
+
+#: src/view/com/modals/AddAppPasswords.tsx:90
+msgid "Please enter a name for your app password. All spaces is not allowed."
+msgstr "請輸入應用程式專用密碼的名稱。不允許使用所有空格。"
```
```suggestion
msgstr "請輸入應用程式專用密碼的名稱。所有空格均不允許使用。"
```

pan93412 commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4527 @@
+#: src/view/com/modals/AppealLabel.tsx:72
+#: src/view/com/modals/AppealLabel.tsx:75
+msgid "Please tell us why you think this content warning was incorrectly applied!"
+msgstr "請告訴我們你認為這個內容警告是錯誤的原因！"
```
```suggestion
msgstr "請告訴我們你認為這個內容警告標示有誤的原因！"
```

(personal suggestion)

pan93412 commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4527 @@
+
+#: src/view/com/modals/SelfLabel.tsx:111
+msgid "Porn"
+msgstr "情色內容"
```
```suggestion
msgstr "色情內容"
```

auroursa replied:
I noticed that this string has been repeatedly modified, previously changed from "色情內容" to "情色內容" here.

pan93412 replied:
Well, then I think it is better to leave it as it currently is.

pan93412 commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4527 @@
+
+#: src/view/com/util/Selector.tsx:107
+msgid "Select option {i} of {numItems}"
+msgstr "從 {i} 項中選擇 {numItems} 項"
```
```suggestion
msgstr "選擇 {numItems} 個項目中的第 {i} 項"
```

https://github.com/bluesky-social/social-app/blob/43b447e5f4345bd6c3a6957cb5aa1587324799e8/src/view/com/util/Selector.tsx#L99-L120

snowleo208 commented:
> I have adjusted all the translation based on the review suggestions, and I believe we are ready to go.

Agree. Looks good to me!

pan93412 commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4527 @@
+
+#: src/view/com/modals/ContentFilteringSettings.tsx:306
+msgid "Set {value} for {labelGroup} content moderation policy"
+msgstr "為 {labelGroup} 內容審核政策設定 {value}"
```
```suggestion
msgstr "將 {labelGroup} 內容審核政策設為 {value}"
```

https://github.com/bluesky-social/social-app/blob/43b447e5f4345bd6c3a6957cb5aa1587324799e8/src/view/com/modals/ContentFilteringSettings.tsx#L293-L313

toto6038 commented:
Hello, @pan93412 forwarded the issue to me and I would like to help proofread the translation and just spot some problems. Will submit suggestions soon 🙏🏿

pan93412 commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4527 @@
+
+#: src/view/com/auth/login/ForgotPasswordForm.tsx:153
+msgid "Sets email for password reset"
+msgstr "設定用於重設密碼的電子郵件"
```
```suggestion
msgstr "設定用於重置密碼的電子郵件"
```

All "重置" or all "重設". Unify them.

auroursa replied:
We had previously committed to changing all instances of "重置" to "重設," so I chose "重設" here (and elsewhere).

pan93412 commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4527 @@
+#: src/view/com/auth/create/Step1.tsx:78
+#: src/view/com/auth/login/LoginForm.tsx:148
+msgid "Sets server for the Bluesky client"
+msgstr "設定 Bluesky 客戶端的伺服器"
```
```suggestion
msgstr "設定 Bluesky 用戶端的伺服器"
```

https://zh.wikipedia.org/zh-tw/客户端

![CleanShot 2024-02-09 at 16 13 00@2x](https://github.com/bluesky-social/social-app/assets/28441561/977bc2ff-b9ae-41c8-993b-aefcd965641d)

pan93412 commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4527 @@
+
+#: src/view/screens/PreferencesHomeFeed.tsx:70
+msgid "Show replies with at least {value} {0}"
+msgstr "顯示至少包含 {value} {0} 的回覆"
```
```suggestion
msgstr "顯示至少包含 {value} 個{0}的回覆"
```

https://github.com/bluesky-social/social-app/blob/43b447e5f4345bd6c3a6957cb5aa1587324799e8/src/view/screens/PreferencesHomeFeed.tsx#L66-L75

pan93412 commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4527 @@
+
+#: src/view/screens/Settings.tsx:276
+msgid "Storage cleared, you need to restart the app now."
+msgstr "已清除存儲，你需要立即重啟應用程式。"
```
```suggestion
msgstr "已清空儲存資料，你需要立即重啟應用程式。"
```

https://github.com/bluesky-social/social-app/pull/2781/files#diff-392e0193d293dc38af7481bd2b758cb6882f26ed36875ba487ad969a3cafd9b6R653

pan93412 commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4527 @@
+#: src/view/com/modals/SwitchAccount.tsx:98
+#: src/view/screens/Settings.tsx:138
+msgid "Switches the account you are logged in to"
+msgstr "切換你登錄的帳號"
```
```suggestion
msgstr "切換你登入的帳號"
```

pan93412 commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4527 @@
+#: src/view/com/modals/AppealLabel.tsx:70
+#: src/view/com/modals/report/InputIssueDetails.tsx:51
+msgid "Text input field"
+msgstr "文字輸入欄"
```
```suggestion
msgstr "文字輸入框"
```

pan93412 commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4527 @@
+
+#: src/view/screens/Support.tsx:36
+msgid "The support form has been moved. If you need help, please <0/> or visit {HELP_DESK_URL} to get in touch with us."
+msgstr "支援表單已移動。如果你需要幫助，請點擊<0/>或訪問 {HELP_DESK_URL} 與我們聯繫。"
```
```suggestion
msgstr "支援表單已移動。如果你需要幫助，請點擊<0/>或進到 {HELP_DESK_URL} 與我們聯絡。"
```

toto6038 replied:
```suggestion
msgstr "支援表單已移至別處。如果需協助，請點擊<0/>或前往 {HELP_DESK_URL} 與我們聯繫。"
```

auroursa replied:
I personally lean towards choosing the translation provided by @toto6038 for this string.

pan93412 commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4527 @@
+
+#: src/view/com/util/moderation/ScreenHider.tsx:83
+msgid "This account has requested that users sign in to view their profile."
+msgstr "此帳號要求使用者登錄後才能查看其個人資料。"
```
```suggestion
msgstr "此帳號要求使用者登入後才能查看其個人資料。"
```

pan93412 commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4527 @@
+#: src/view/shell/desktop/RightNav.tsx:137
+#: src/view/shell/Drawer.tsx:660
+msgid "Your invite codes are hidden when logged in using an App Password"
+msgstr "在使用應用程式專用密碼登錄時，你的邀請碼將被隱藏"
```
```suggestion
msgstr "在使用應用程式專用密碼登入時，你的邀請碼將被隱藏"
```

pan93412 commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4527 @@
+#: src/view/com/auth/login/Login.tsx:158
+#: src/view/com/auth/login/PasswordUpdatedForm.tsx:31
+msgid "You can now sign in with your new password."
+msgstr "你現在可以使用新密碼登錄."
```
```suggestion
msgstr "你現在可以使用新密碼登入。"
```

pan93412 commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4527 @@
+
+#: src/view/com/modals/AddAppPasswords.tsx:155
+msgid "Use this to sign into the other app along with your handle."
+msgstr "使用這個和你的帳號代碼一起登錄其他應用。"
```
```suggestion
msgstr "使用這個和你的帳號代碼一起登入其他應用程式。"
```

pan93412 commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4527 @@
+
+#: src/view/com/auth/create/state.ts:300
+msgid "Unfortunately, you do not meet the requirements to create an account."
+msgstr "很遺憾，你不符合創建帳號的要求。"
```
```suggestion
msgstr "很遺憾，你不符合建立帳號的要求。"
```

pan93412 commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4527 @@
+
+#: src/view/com/util/forms/PostDropdownBtn.tsx:192
+msgid "This will hide this post from your feeds."
+msgstr "這將在你的消息流中隱藏此貼文。"
```
```suggestion
msgstr "這將在你的訊息流中隱藏此貼文。"
```

pan93412 commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4527 @@
+
+#: src/view/screens/AppPasswords.tsx:195
+msgid "Use app passwords to login to other Bluesky clients without giving full access to your account or password."
+msgstr "使用應用程式專用密碼登錄到其他 Bluesky 客戶端，而無需提供你的帳號或密碼。"
```
```suggestion
msgstr "使用應用程式專用密碼登入到其他 Bluesky 用戶端，而無需提供你的帳號或密碼。"
```

pan93412 commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4527 @@
+
+#: src/view/com/modals/ServerInput.tsx:105
+msgid "Use your domain as your Bluesky client service provider"
+msgstr "使用你的域名作為 Bluesky 客戶端服務提供商"
```
```suggestion
msgstr "將你的域名用作 Bluesky 用戶端服務提供商"
```

auroursa replied:
This is a place we overlooked before. We had replaced all instances of "域名" with "網域", so I made a few small modifications.

pan93412 commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4527 @@
+
+#: src/view/com/profile/ProfileHeader.tsx:701
+msgid "View {0}'s avatar"
+msgstr "查看{0}的頭像"
```
```suggestion
msgstr "查看{0}的頭貼"
```

pan93412 commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4527 @@
+
+#: src/view/com/modals/LinkWarning.tsx:75
+msgid "Visit Site"
+msgstr "訪問網站"
```
```suggestion
msgstr "造訪網站"
```

pan93412 commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4527 @@
+
+#: src/screens/Onboarding/StepAlgoFeeds/index.tsx:124
+msgid "We also think you'll like \"For You\" by Skygaze:"
+msgstr "我們認為你還會喜歡 Skygaze 維護的「For You」:"
```
```suggestion
msgstr "我們認為你還會喜歡 Skygaze 維護的「For You」："
```

pan93412 commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4527 @@
+
+#: src/view/screens/ProfileList.tsx:85
+msgid "We're sorry, but we were unable to resolve this list. If this persists, please contact the list creator, @{handleOrDid}."
+msgstr "很抱歉，我們無法解析此列表。如果問題持續發生，請聯繫列表創建者，@{handleOrDid}。"
```
```suggestion
msgstr "很抱歉，我們無法解析此列表。如果問題持續發生，請聯繫列表建立者 @{handleOrDid}。"
```

pan93412 commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4527 @@
+
+#: src/view/screens/Search/Search.tsx:253
+msgid "We're sorry, but your search could not be completed. Please try again in a few minutes."
+msgstr "很抱歉，無法完成你的搜索。請稍後再試。"
```
```suggestion
msgstr "很抱歉，無法完成你的搜尋請求。請稍後再試。"
```

pan93412 commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4527 @@
+
+#: src/view/com/modals/lang-settings/ContentLanguagesSettings.tsx:77
+msgid "Which languages would you like to see in your algorithmic feeds?"
+msgstr "你想在算法訊息流中看到哪些語言？"
```
```suggestion
msgstr "你想在演算法訊息流中看到哪些語言？"
```

pan93412 commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4527 @@
+
+#: src/view/com/auth/login/SetNewPasswordForm.tsx:107
+msgid "You will receive an email with a \"reset code.\" Enter that code here, then enter your new password."
+msgstr "你將收到一封包含重置碼的電郵。請在此輸入該重置代碼，然後輸入你的新密碼。"
```
```suggestion
msgstr "你將收到一封包含重置碼的電子郵件。請在此輸入該重置代碼，然後輸入你的新密碼。"
```

pan93412 commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4527 @@
+"Report-Msgid-Bugs-To: \n"
+"PO-Revision-Date: \n"
+"Last-Translator: Frudrax Cheng <i@cynosura.one>\n"
+"Language-Team: Frudrax Cheng, Kuwa Lee, noeFly, snowleo208, Kisaragi Hiu\n"
```
```suggestion
"Language-Team: Frudrax Cheng, Kuwa Lee, noeFly, snowleo208, Kisaragi Hiu, Yi-Jyun Pan\n"
```

Provide my preferred nickname as others do.

pan93412 reviewed (commented):
These suggestions mainly fix up:

- Some translation were translated without a correct context. I corrected it according to the hints in source code.
- Some terms are not unified. I point these out.
- Some terms are not common in Taiwan. I correct it with the terms from applications that Taiwanese usually use or the authoritative dictionaries.

I have tried my best to provide the context and references as much as possible. Feel free to write your comments of these suggestions here.

auroursa commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4527 @@
+
+#: src/view/screens/Settings.tsx:529
+msgid "Opens the home feed preferences"
+msgstr "開啟首頁訊息流偏好"
```
"開啟首頁訊息流設定偏好" seems to enhance readability.

toto6038 commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4527 @@
+
+#: src/view/com/auth/login/ForgotPasswordForm.tsx:103
+msgid "Enter the email you used to create your account. We'll send you a \"reset code\" so you can set a new password."
+msgstr "輸入你用於創建帳號的電子郵件。我們將向你發送重設碼，以便你設定新密碼。"
```
```suggestion
msgstr "輸入你建立帳號時用的電子信箱。我們會寄送重設碼給你，以便設定新密碼。"
```

toto6038 commented:
As my concern is mostly addressed in @pan93412's suggestions, further feedbacks are already included there.

auroursa commented:
@pan93412 Thank you for your hard work. I have merged all uncontroversial modification suggestions.

I've organized the remaining tasks, and I may need more feedback:
- Choosing the translation for "build version".
- Deciding between "色情內容" or "情色內容".
- Improving the translation for "Copies".
- Line 2507 seems to need improvement.

KuwaLee commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4527 @@
+
+#: src/view/com/notifications/FeedItem.tsx:348
+msgid "Opens an expanded list of users in this notification"
+msgstr "開啟此通知中的擴展使用者列表"
```
"展開通知中的使用者列表"？

toto6038 commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4527 @@
+
+#: src/view/screens/Settings.tsx:841
+msgid "Build version {0} {1}"
+msgstr "構建版本號 {0} {1}"
```
I propose to have a poll here.
😄: MS
🎉: FOSS
🚀: Apple

I believe that Apple's convention is proper enough.

toto6038 commented:
To collect reviewers' opinion toward the translation of "Build version". I opened a poll here ⬇️ 
https://github.com/bluesky-social/social-app/pull/2781#discussion_r1484078311

kisaragi-hiu commented:
- "build version": All seem fine to me though I lean towards 建構版本號 to avoid colliding with "version number". (Hence my vote.)
- "色情內容" or "情色內容": either seem fine with me. 色情 is explicit and perhaps has negative connotations (just like English "pornography"), while 情色 is less negative and more eloquent, but both get the point across.
- "Copies <thing>" in accessibility text: I suggested adding a 會 to indicate that's an action to be taken - this is a little clumsy though. I'm fine with it removed, as pan93412 suggested.
- Line 2507 (expanded list of users in notification): I suggest 展開此通知的使用者列表, which gets the point across without having to describe both that the notification has a list of users and that the list is expandable (and this expands it) in the same noun phrase.

KuwaLee commented:
I agree with @kisaragi-hiu

auroursa commented:
I have consolidated the voting results and comments and submitted the modifications.

- I have chosen the FOSS-style term 'Build Version Number' for '建構版本號'."
- In this PR, I will keep "情色內容" unchanged.
- "Copies" has been revised by removing "會."
- Line 2507 -> “展開此通知的使用者列表”

Thanks for the responsible reviews, I believe our translation has reached an unprecedented level of quality. Sorry for introducing quite a few simplified Chinese terms in my initial translations, it was unintentional. Once again, thanks to everyone for your contributions.

auroursa commented:
@pfrazee I am confident that we have reached a consensus and ready to go.

auroursa commented:
I made some minor modifications: In the previous discussions, we translated "Yes" and "No" as "開" and "關" respectively. However, some descriptions were not synchronized. I have fixed it.
![image](https://github.com/bluesky-social/social-app/assets/46548406/4d3a68b2-71d1-48f0-9d7c-4f7f459ad222)

snowleo208 commented:
@auroursa Is this PR ready to go? I think we already reached consensus and only need an approval now, right? :smile:

auroursa commented:
@snowleo208 Yeah I believe this PR is ready to go and awaiting final approval.
Don’t worry, the developers will handle it well. They just have more pressing matters to attend to first.

auroursa commented:
@pan93412 @toto6038 @kisaragi-hiu @snowleo208 
Everyone, I've completed merging the strings from the upstream. Please take a look at commits [22278e1](https://github.com/bluesky-social/social-app/pull/2781/commits/22278e1ed4b1079051eaf1d95886f6c06c505da2) & [cfc9b4d](https://github.com/bluesky-social/social-app/pull/2781/commits/cfc9b4dc2b4e77be8dc54201f9fb515cabc2ef0b) and help me reviewing these new translations. Thank you!

toto6038 commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -4509,7 +4509,7 @@ msgstr "你的帳號已刪除"
 #: src/view/screens/Settings/ExportCarDialog.tsx:47
 msgid "Your account repository, containing all public data records, can be downloaded as a \"CAR\" file. This file does not include media embeds, such as images, or your private data, which must be fetched separately."
-msgstr ""
+msgstr "你的帳號存放庫，包含所有公開的資料記錄，可以下載為一個「CAR」文件。該文件不包括嵌入的媒體，如圖片，或你的私人資料，目前這些資料必須單獨擷取。"
```
```suggestion
msgstr "你可以將你的帳號存放庫下載為一個「CAR」檔案。該檔案包含了所有公開的資料紀錄，但不包括嵌入媒體，例如圖片或你的私人資料，目前這些資料必須另外擷取。
```

toto6038 commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -3933,7 +3933,7 @@ msgstr "沒有 Bluesky 帳號，無法查看此內容。"
 #: src/view/screens/Settings/ExportCarDialog.tsx:75
 msgid "This feature is in beta. You can read more about repository exports in <0>this blogpost.</0>"
-msgstr ""
+msgstr "此功能目前為測試版本。您可以在<0>這篇部落格文章</0>中了解更多有關存放庫匯出的資訊"
```
```suggestion
msgstr "此功能目前為測試版本。您可以在<0>這篇部落格文章</0>中了解更多有關匯出存放庫的資訊"
```

toto6038 reviewed (commented):
The batch of suggestions include:

- Correct the usage of word
- Rewrite sentences in hope to be more fluent

toto6038 commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4587 @@
+"Report-Msgid-Bugs-To: \n"
+"PO-Revision-Date: \n"
+"Last-Translator: Frudrax Cheng <i@cynosura.one>\n"
+"Language-Team: Frudrax Cheng, Kuwa Lee, noeFly, snowleo208, Kisaragi Hiu, Yi-Jyun Pan\n"
```
```suggestion
"Language-Team: Frudrax Cheng, Kuwa Lee, noeFly, snowleo208, Kisaragi Hiu, Yi-Jyun Pan, toto6038\n"
```

toto6038 replied:
Provide my preferred nickname as @pan93412 and others do.

KuwaLee commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -3288,7 +3288,7 @@ msgstr "在下面選擇一些要跟隨的帳號"
 #: src/view/com/auth/server-input/index.tsx:82
 msgid "Select the service that hosts your data."
-msgstr ""
+msgstr "選擇托管你資料的服務"
```
託管

auroursa commented:
After compiling, I checked each part, and it works well.

snowleo208 commented:
Look good to me :smiley:

auroursa commented:
@pfrazee I have merged all changes from #2879, and it works well in the latest version. I think we are ready to go.

Ge072 commented:
> Traditional Chinese is widely utilized in Taiwan, Hong Kong, and Macau, boasting a user base of nearly 30 million globally. For this project, I took the traditional Chinese from Taiwan as the basis for creating the messages.po file.
> 
> I humbly seek additional feedback from the community users, thank you.

toto6038 commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -1370,7 +1370,7 @@ msgstr "輸入你的使用者名稱和密碼"
 #: src/view/com/auth/create/Step3.tsx:67
 msgid "Error receiving captcha response."
-msgstr ""
+msgstr "Captcha 給出了錯誤的回應。"
```
```suggestion
msgstr "取得 Captcha 回應時發生錯誤。"
```

auroursa replied:
Looks good to me

auroursa commented:
Looks great so far.

auroursa commented:
I have fixed it based on the review suggestions.

auroursa commented:
A small translation correction, this should not be "圖書舘"
I will change it to "圖片庫"
![image](https://github.com/bluesky-social/social-app/assets/46548406/5f5164db-5dcc-4a47-a6bd-d750e8d14aae)

snowleo208 commented:
This is ready to go, right? Since this PR already hanging around for a few weeks, I think we should prioritize pushing this PR live first and ignore upstream changes for now. 

If there's any upstream changes, we can always create another PR for that. May we ask for review again from bluesky team?

cirx1e commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4613 @@
+
+#: src/view/com/modals/ContentFilteringSettings.tsx:141
+msgid "Adult content can only be enabled via the Web at <0/>."
+msgstr "成人内容只能在網頁上<0/>啟用。"
```
```suggestion
msgstr "成人內容只能在網頁上<0/>啟用。"
```

「`内`」->「`內`」

cirx1e commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4613 @@
+
+#: src/view/screens/Feeds.tsx:666
+msgid "All the feeds you've saved, right in one place."
+msgstr "你已儲存的所有訊息流都集中在一处。"
```
```suggestion
msgstr "你已儲存的所有訊息流都集中在一處。"
```

「处」->「處」

cirx1e commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4613 @@
+#: src/view/com/auth/login/ForgotPasswordForm.tsx:221
+#: src/view/com/modals/ChangePassword.tsx:168
+msgid "Already have a code?"
+msgstr "已經有重設碼了？"
```
```suggestion
msgstr "已經有驗證碼了？"
```

「重設碼」->「驗證碼」

cirx1e commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4613 @@
+
+#: src/view/com/modals/ChangeEmail.tsx:119
+msgid "An email has been sent to your previous address, {0}. It includes a confirmation code which you can enter below."
+msgstr "一封電子郵件已發送至先前填寫的電郵地址 {0}。請查閱郵件並在下方輸入驗證碼。"
```
```suggestion
msgstr "一封電子郵件已發送至先前填寫的電子郵件地址 {0}。請查閱郵件並在下方輸入驗證碼。"
```

cirx1e commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4613 @@
+
+#: src/view/com/modals/DeleteAccount.tsx:169
+msgid "Check your inbox for an email with the confirmation code to enter below:"
+msgstr "查看寄送至你電子郵箱的確認郵件，然後在下方輸入收到的驗證碼："
```
```suggestion
msgstr "查看寄送至你電子信箱的確認郵件，然後在下方輸入收到的驗證碼："
```

台灣應該較常使用 "電子信箱" 及 "電子郵件地址"？

auroursa replied:
I will use "電子郵件地址" consistently throughout the context.

cirx1e commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4613 @@
+#: src/view/com/modals/ChangeEmail.tsx:141
+#: src/view/com/modals/Waitlist.tsx:88
+msgid "Email"
+msgstr "電子郵件"
```
```suggestion
msgstr "電子信箱"
```

auroursa replied:
I will refrain from making changes to "電子郵件" for now, as I have not yet decided how to differentiate between "email" and "email address". "電子信箱地址" seems to be uncommonly used in Taiwan.

cirx1e commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4613 @@
+#: src/view/com/modals/ChangeEmail.tsx:56
+#: src/view/com/modals/ChangeEmail.tsx:88
+msgid "Email updated"
+msgstr "電子郵件已更新"
```
```suggestion
msgstr "電子信箱已更新"
```

cirx1e commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4613 @@
+
+#: src/view/com/modals/ChangeEmail.tsx:111
+msgid "Email Updated"
+msgstr "電子郵件已更新"
```
```suggestion
msgstr "電子信箱已更新"
```

cirx1e commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4613 @@
+
+#: src/view/com/modals/VerifyEmail.tsx:78
+msgid "Email verified"
+msgstr "電子郵件已驗證"
```
```suggestion
msgstr "電子信箱已驗證"
```

cirx1e commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4613 @@
+
+#: src/view/screens/Settings/index.tsx:312
+msgid "Email:"
+msgstr "電子郵件："
```
```suggestion
msgstr "電子信箱："
```

cirx1e commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4613 @@
+
+#: src/view/com/auth/login/ForgotPasswordForm.tsx:107
+msgid "Enter the email you used to create your account. We'll send you a \"reset code\" so you can set a new password."
+msgstr "輸入你用於建立帳號的電子郵件。我們將向你發送重設碼，以便你設定新密碼。"
```
```suggestion
msgstr "輸入你用於建立帳號的電子信箱。我們將向你發送驗證碼，以便你設定新密碼。"
```

cirx1e commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4613 @@
+
+#: src/view/com/modals/Waitlist.tsx:78
+msgid "Enter your email"
+msgstr "輸入你的電子郵箱"
```
```suggestion
msgstr "輸入你的電子信箱"
```

cirx1e commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4613 @@
+
+#: src/view/com/auth/create/Step1.tsx:195
+msgid "Enter your email address"
+msgstr "輸入你的電子郵箱地址"
```
```suggestion
msgstr "輸入你的電子郵件地址"
```

cirx1e commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4613 @@
+
+#: src/view/com/modals/ChangeEmail.tsx:41
+msgid "Enter your new email above"
+msgstr "請在上方輸入你的新電子郵箱"
```
```suggestion
msgstr "請在上方輸入你的新電子信箱"
```

cirx1e commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4613 @@
+
+#: src/view/com/modals/ChangeEmail.tsx:117
+msgid "Enter your new email address below."
+msgstr "請在下方輸入你的新電子郵箱。"
```
```suggestion
msgstr "請在下方輸入你的新電子信箱。"
```

cirx1e commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4613 @@
+
+#: src/view/com/util/forms/PostDropdownBtn.tsx:88
+msgid "Failed to delete post, please try again"
+msgstr "無法刪除貼文，请重試"
```
```suggestion
msgstr "無法刪除貼文，請重試"
```

「请」->「請」

cirx1e commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4613 @@
+
+#: src/view/com/auth/login/SetNewPasswordForm.tsx:138
+msgid "Input code sent to your email for password reset"
+msgstr "輸入發送到你電子郵箱的重設碼以重設密碼"
```
```suggestion
msgstr "輸入發送到你電子信箱的驗證碼以重設密碼"
```

cirx1e commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4613 @@
+
+#: src/view/com/auth/create/Step1.tsx:200
+msgid "Input email for Bluesky account"
+msgstr "輸入 Bluesky 帳號的電子郵箱"
```
```suggestion
msgstr "輸入 Bluesky 帳號的電子信箱"
```

cirx1e commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4613 @@
+
+#: src/view/com/auth/login/LoginForm.tsx:197
+msgid "Input the username or email address you used at signup"
+msgstr "輸入註冊時使用的使用者名稱或電子郵箱"
```
```suggestion
msgstr "輸入註冊時使用的使用者名稱或電子信箱"
```

cirx1e commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4613 @@
+
+#: src/view/com/modals/Waitlist.tsx:90
+msgid "Input your email to get on the Bluesky waitlist"
+msgstr "輸入你的電子郵箱以加入 Bluesky 候補列表"
```
```suggestion
msgstr "輸入你的電子信箱以加入 Bluesky 候補列表"
```

cirx1e commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4613 @@
+#: src/view/com/auth/onboarding/WelcomeDesktop.tsx:72
+#: src/view/com/auth/onboarding/WelcomeMobile.tsx:72
+msgid "Never lose access to your followers and data."
+msgstr "永遠不會失去對你的追隨者和資料的存取權。"
```
```suggestion
msgstr "永遠不會失去對你的跟隨者和資料的存取權。"
```

cirx1e commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4613 @@
+
+#: src/screens/Onboarding/StepFinished.tsx:119
+msgid "Never lose access to your followers or data."
+msgstr "永遠不會失去對你的追隨者或資料的存取權。"
```
```suggestion
msgstr "永遠不會失去對你的跟隨者或資料的存取權。"
```

cirx1e commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4613 @@
+
+#: src/view/com/modals/ChangeEmail.tsx:67
+msgid "Please confirm your email before changing it. This is a temporary requirement while email-updating tools are added, and it will soon be removed."
+msgstr "更改前請先確認你的電子郵箱。這是新增電子郵箱更新工具的臨時要求，此限制將很快被移除。"
```
```suggestion
msgstr "更改前請先確認你的電子信箱。這是新增電子信箱更新工具的臨時要求，此限制將很快被移除。"
```

cirx1e commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4613 @@
+
+#: src/view/com/auth/create/state.ts:103
+msgid "Please enter your email."
+msgstr "請輸入你的電子郵件。"
```
```suggestion
msgstr "請輸入你的電子信箱。"
```

cirx1e commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4613 @@
+
+#: src/view/com/modals/VerifyEmail.tsx:101
+msgid "Please Verify Your Email"
+msgstr "請驗證你的電子郵箱"
```
```suggestion
msgstr "請驗證你的電子信箱"
```

cirx1e commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4613 @@
+
+#: src/view/screens/Settings/index.tsx:949
+msgid "Protect your account by verifying your email."
+msgstr "通過驗證電子郵件來保護你的帳號。"
```
```suggestion
msgstr "通過驗證電子信箱來保護你的帳號。"
```

cirx1e commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4613 @@
+#: src/view/com/auth/login/SetNewPasswordForm.tsx:124
+#: src/view/com/auth/login/SetNewPasswordForm.tsx:136
+msgid "Reset code"
+msgstr "重設碼"
```
```suggestion
msgstr "驗證碼"
```

cirx1e commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4613 @@
+
+#: src/view/com/modals/ChangePassword.tsx:190
+msgid "Reset Code"
+msgstr "重設碼"
```
```suggestion
msgstr "驗證碼"
```

cirx1e commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4613 @@
+
+#: src/view/com/modals/Waitlist.tsx:51
+msgid "Something went wrong. Check your email and try again."
+msgstr "發生了一些問題。請檢查你的電子郵件，然後重試。"
```
```suggestion
msgstr "發生了一些問題。請檢查你的電子信箱，然後重試。"
```

cirx1e commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4613 @@
+
+#: src/view/com/modals/VerifyEmail.tsx:119
+msgid "This is important in case you ever need to change your email or reset your password."
+msgstr "這很重要，以防你將來需要更改電子郵箱或重設密碼。"
```
```suggestion
msgstr "這很重要，以防你將來需要更改電子信箱或重設密碼。"
```

cirx1e commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4613 @@
+#: src/view/com/profile/FollowButton.tsx:55
+msgctxt "action"
+msgid "Unfollow"
+msgstr "取消關注"
```
```suggestion
msgstr "取消跟隨"
```

cirx1e commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4613 @@
+
+#: src/view/com/profile/ProfileHeader.tsx:484
+msgid "Unfollow {0}"
+msgstr "取消關注 {0}"
```
```suggestion
msgstr "取消跟隨 {0}"
```

cirx1e commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4613 @@
+
+#: src/view/screens/Settings/index.tsx:910
+msgid "Verify email"
+msgstr "驗證電子郵件"
```
```suggestion
msgstr "驗證電子信箱"
```

cirx1e commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4613 @@
+
+#: src/view/screens/Settings/index.tsx:935
+msgid "Verify my email"
+msgstr "驗證我的電子郵件"
```
```suggestion
msgstr "驗證我的電子信箱"
```

cirx1e commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4613 @@
+
+#: src/view/screens/Settings/index.tsx:944
+msgid "Verify My Email"
+msgstr "驗證我的電子郵件"
```
```suggestion
msgstr "驗證我的電子信箱"
```

cirx1e commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4613 @@
+#: src/view/com/modals/ChangeEmail.tsx:205
+#: src/view/com/modals/ChangeEmail.tsx:207
+msgid "Verify New Email"
+msgstr "驗證新的電子郵件"
```
```suggestion
msgstr "驗證新的電子信箱"
```

cirx1e commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4613 @@
+
+#: src/view/com/modals/VerifyEmail.tsx:103
+msgid "Verify Your Email"
+msgstr "驗證你的電子郵件"
```
```suggestion
msgstr "驗證你的電子信箱"
```

cirx1e commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4613 @@
+
+#: src/view/com/auth/login/SetNewPasswordForm.tsx:107
+msgid "You will receive an email with a \"reset code.\" Enter that code here, then enter your new password."
+msgstr "你將收到一封包含重設碼的電子郵件。請在此輸入該重設代碼，然後輸入你的新密碼。"
```
```suggestion
msgstr "你將收到一封包含驗證碼的電子郵件。請在此輸入該代碼，然後輸入你的新密碼。"
```

cirx1e commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4613 @@
+#: src/view/com/auth/login/ForgotPasswordForm.tsx:70
+#: src/view/com/modals/ChangePassword.tsx:54
+msgid "Your email appears to be invalid."
+msgstr "你的電子郵箱似乎無效。"
```
```suggestion
msgstr "你的電子信箱似乎無效。"
```

cirx1e commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4613 @@
+
+#: src/view/com/modals/Waitlist.tsx:109
+msgid "Your email has been saved! We'll be in touch soon."
+msgstr "你的電子郵箱已儲存！我們將很快聯繫你。"
```
```suggestion
msgstr "你的電子信箱已儲存！我們將很快聯繫你。"
```

cirx1e commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4613 @@
+
+#: src/view/com/modals/ChangeEmail.tsx:125
+msgid "Your email has been updated but not verified. As a next step, please verify your new email."
+msgstr "你的電子郵箱已更新但尚未驗證。作為下一步，請驗證你的新電子郵箱。"
```
```suggestion
msgstr "你的電子信箱已更新但尚未驗證。作為下一步，請驗證你的新電子信箱。"
```

cirx1e commented on src/locale/locales/zh-TW/messages.po:
```diff
@@ -0,0 +1,4613 @@
+
+#: src/view/com/modals/VerifyEmail.tsx:114
+msgid "Your email has not yet been verified. This is an important security step which we recommend."
+msgstr "你的電子郵箱尚未驗證。這是一個我們建議的重要安全步驟。"
```
```suggestion
msgstr "你的電子信箱尚未驗證。這是一個我們建議的重要安全步驟。"
```

cirx1e reviewed (commented):
一些小建議
- 部分簡體字未轉換
- 台灣應該較常使用 "電子信箱" 及 "電子郵件地址"？
- "follow" 統一譯為「跟隨」
- 「重設碼」->「驗證碼」

auroursa commented:
I appreciate your patience and contributions. However, it seems that upstream has temporarily paused merging translations. I will merge your contributions into the .po files once translation merging is reopened.

pfrazee commented:
Okay sorry everybody, I'm back in action for handling translation PRs. @auroursa how do you want to proceed -- should we merge as-is or do you want to factor in more updates?

auroursa commented:
I have merged the review suggestions and the upstream updates. And I believe additional translation updates should be handled in a new PR, as this one has become too lengthy. So I think we are ready to go. @pfrazee

auroursa commented:
@cirx1e I haven't figured out how to differentiate between "email" and "email address" yet. It seems a bit odd to translate "email address" as "電子信箱地址". I've merged the other review suggestions into the latest .po file.
I'd like to gather input from other translators and address it in the next translation PR. This PR has been open for too long, and I'd like to push for its merge as soon as possible. Thank you for your hard work.
